web_src/js/features/repo-issue-list.js

```diff
@@ -40,7 +40,7 @@ function initRepoIssueListCheckboxes() {
     el.addEventListener('change', syncIssueSelectionState);
   }
 
-  issueSelectAll.addEventListener('change', () => {
+  issueSelectAll?.addEventListener('change', () => {
```

Comment:
This is not the only place accessing `issueSelectAll` ...........

Reply:
The function is big and does more. Need to check if the other code is needed or not.

Reply:
Also I guess there must be a cleaner way to determine logged in state in JS.

Reply:
> The function is big and does more. Need to check if the other code is needed or not.

It is not that big. All it does is to handle the checkbox states. It is called as "initRepoIssueListCheckboxes", I guess there should be no checkbox if the user isn't signed in.

Reply:
Yes, the whole function can be skipped for logged out users.

I've  added `window.config.isSignedIn` in https://github.com/go-gitea/gitea/pull/29854/commits/c357b98a6d40f0daa344db26cb5b7084deb48f34 now.